src/library/Uno.Themes.WinUI.Markup/Theme/ToggleButton.cs

```diff
@@ -0,0 +1,379 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushCheckedPressed")]
+						public static ResourceValue<Brush> Pressed => new("IconToggleButtonBorderBrushCheckedPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushCheckedDisabled")]
+						public static ResourceValue<Brush> Disabled => new("IconToggleButtonBorderBrushCheckedDisabled", true);
+					}
+
+					public static class BorderBrushIndeterminate
+					{
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminate")]
+						public static ResourceValue<Brush> Default => new("IconToggleButtonBorderBrushIndeterminate", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminatePointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("IconToggleButtonBorderBrushIndeterminatePointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminatePressed")]
+						public static ResourceValue<Brush> Pressed => new("IconToggleButtonBorderBrushIndeterminatePressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminateDisabled")]
+						public static ResourceValue<Brush> Disabled => new("IconToggleButtonBorderBrushIndeterminateDisabled", true);
+					}
+
+					public static class StateCircle
+					{
+						public static class Fill
+						{
+							[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonStateCircleFill")]
+							public static ResourceValue<Brush> Default => new("IconToggleButtonStateCircleFill", true);
+						}
+
+						public static class Opacity
+						{
+							[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityPointerOver")]
+							public static ResourceValue<double> PointerOver => new("IconToggleButtonStateCircleOpacityPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityPressed")]
+							public static ResourceValue<double> Pressed => new("IconToggleButtonStateCircleOpacityPressed", true);
+
+							[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityFocused")]
+							public static ResourceValue<double> Focused => new("IconToggleButtonStateCircleOpacityFocused", true);
+						}
+					}
+
+					public static class Typography
+					{
+						[ResourceKeyDefinition(typeof(Thickness), "IconToggleButtonBorderThickness")]
+						public static ResourceValue<Thickness> BorderThickness => new("IconToggleButtonBorderThickness", true);
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonMinWidth")]
+						public static ResourceValue<double> MinWidth => new("IconToggleButtonMinWidth", true);
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonMinHeight")]
+						public static ResourceValue<double> MinHeight => new("IconToggleButtonMinHeight", true);
```

Comment:
Same comment as above, not in a `Typography` class

Reply:
updated